JenkinsfileRT

```diff
@@ -0,0 +1,22 @@
+// Obtain files from source control system.
+if (utils.scm_checkout()) return
+
+// Define each build configuration, copying and overriding values as necessary.
+bc0 = new BuildConfig()
+bc0.nodetype = "linux-stable"
+bc0.name = "debug"
+bc0.build_cmds = ["conda env update --file=environment.yml",
+	       	  "with_env -n jwql python setup.py install"]
+bc0.test_cmds = ["cp /grp/jwst/ins/jwql/config.json jwql/utils/",
```

Comment:
@SaOgaz Is there any way we could import this filepath from the `config.json` file or must we hard-code this here?

Reply:
@SaOgaz If that is possible, we should do that, since this exposes how the internal filesystem is structured and thus is a security risk.  

Another thing we could do is to create a symbolic link in the `$HOME` directory that points to the location of the `config.json` file.